docs/sftt/sftt-setup-backend.md

```diff
@@ -0,0 +1,32 @@
+# Setting up the Backend
+This is a comprehensive overview of the steps required and everything needed to run the database locally. 
+
+### Downloads
+1. Clone the [Github repo](https://github.com/Code-4-Community/speak-for-the-trees-backend)
+2. Download [PostgreSQL](https://www.postgresql.org/download/) (Make sure it comes with pgAdmin4)
+3. Download [Java JDK 8](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html)
```

Comment:
Maybe include steps here as sub-bullets to item 3 on how to set the JAVA_HOME environment variable and also confirm that they've set it properly? Like by checking in the command line. Then you could also remove item 4 in "Installing Maven". Makes a bit more sense to me.